src/common/tiles/BigTile/styled.js

```diff
@@ -146,6 +182,7 @@ export const Genre = styled.p`
   display: flex;
```

Comment:
U mnie te Genres, wygladają i zachowują się jak zwykłe linki. Myślę że można im zabrać `text-decoration`

Reply:
W sumie to nie byla kwestia text-decoration, bo to juz jest ustawione na none dla linkow, tylko kwestia tego, ze GenreItem powinien miec color black a do tej pory nie mial, dobrze, ze to wylapales bo ja to tez widzialem ale nie zapisalem sobie i mi ucieklo :P